presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2883,6 +2884,61 @@ public void testGroupedExecution()
         };
     }
 
+    @Test
+    public void testWriteStagingFiles()
+    {
+        try {
+            Session session = Session.builder(getSession())
+                    .setCatalogSessionProperty(catalog, WRITING_STAGING_FILES_ENABLED, "true")
+                    .build();
+
+            // Test CREATE TABLE AS
+            assertUpdate(
```

Comment:
Do we want to read these tables back and see if the data is expected? -- It looks to me even the rename does nothing, today the tests will still pass ? :P